updater/spec/dependabot/sentry_spec.rb

```diff
@@ -5,6 +5,7 @@
 
 RSpec.describe ExceptionSanitizer do
   let(:message) { "kaboom" }
+  # TODO migrate this from "data" to "event"/"hint"
```

Comment:
If I can figure out how to get access to the debugger this should be straightforward...